lib/wallet/integration/src/Test/Integration/Scenario/API/Shelley/TransactionsNew.hs

```diff
@@ -1220,6 +1221,118 @@ spec = describe "NEW_SHELLEY_TRANSACTIONS" $ do
             , expectField (#witnessCount) (`shouldBe` witnessCountWithNativeScript)
             ]
 
+        let payloadPolicyId = Json [json|{
+                "policy_script_template":
+                    { "all":
+                       [ "cosigner#0"
+                       ]
+                    }
+                }|]
+        let postPolicyId = Link.postPolicyId @'Shelley wa
+        rGet <- request @ApiPolicyId ctx postPolicyId Default payloadPolicyId
+        verify rGet
+            [ expectResponseCode HTTP.status202
+            ]
+        let (ApiPolicyId (ApiT policyId')) = getFromResponse Prelude.id rGet
+
+        eventually "wb wallet has received funds" $ do
+            request @ApiWallet ctx (Link.getWallet @'Shelley wb) Default Empty
```

Comment:
Could you change this to check for the `status` of the transaction instead, like we discussed some time ago for another integration test? (I forgot which one)

I.e. instead of checking whether a balance has increased, we check whether this particular transactions has been accepted into the ledger by querying the [getTransaction][1] endpoint with the transaction id and waiting until the `status` is `in_ledger`.

  [1]: https://cardano-foundation.github.io/cardano-wallet/api/edge/#operation/getTransaction

Reply:
ah, yes. done